src/gui/folderman.cpp

```diff
@@ -734,15 +734,11 @@ QString FolderMan::checkPathValidityRecursive(const QString &path)
         return FolderMan::tr("No valid folder selected!");
     }
 
-#ifdef Q_OS_WIN
-    Utility::NtfsPermissionLookupRAII ntfs_perm;
```

Comment:
Why was the raii removed? It has effects on QFileInfo

Reply:
fixed